### PR TITLE
Added payment view

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Part of #4 

#### What's this PR do?
Adds a view for the payment and tweaks the skeleton UI to make use of it. This view creates an unfulfilled `Order` and returns a form which is submitted to CyberSource. This PR does not include the order fulfillment view, that will be in the next PR.

#### How should this be manually tested?
Create a `Installment` with related a `Klass` and `Bootcamp`. Then log in and type in the klass id into the textbox (this is `klass.klass_id`, not `klass.id`) and a total. Click 'Pay' and you should be directed to an error page on Cybersource. If you go into the database you should see an `Order` that has status `CREATED` and a `Line` which has the `klass_id` you typed in.

### CODE REVIEW

File: ecommerce/views.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        """
+        Create an unfulfilled order and return a response for it.
+        """
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        total = Decimal(serializer.data['total'])
+        klass_id = serializer.data['klass_id']
+
+        order = create_unfulfilled_order(self.request.user, klass_id, total)
+        redirect_url = self.request.build_absolute_uri('/')
```
Comment (giocalitri) [inline ecommerce/views.py]: shouldn't `/` be built as a reverse of an url?

Comment (noisecapella) [inline ecommerce/views.py]: Yes

**Comment by giocalitri** [comment]
Functionality is fine.

**Comment by Ferdi** [comment]
I cant resist , is this "Klass" and not "class"  is suppose to be some kind of a joke ? :-)

File: ecommerce/api.py [hunk 2]
```diff
@@ -2,18 +2,101 @@
...
+            A klass id, a class within a bootcamp
+
+    Returns:
+        Decimal: The total amount paid by a user for a klass
+    """
+    return Line.objects.filter(
+        order__status=Order.FULFILLED,
+        order__user=user,
+        klass_id=klass_id,
+    ).aggregate(price=Sum('price'))['price'] or Decimal(0)
```
Comment (noisecapella) [inline ecommerce/api.py]: This spans multiple `Line`s so I figured a separate function made sense here

File: bootcamp/urls_test.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class URLTests(TestCase):
+    """URL tests"""
+
+    def test_urls(self):
+        """Make sure URLs match with resolved names"""
+        assert reverse('bootcamp-index') == '/'
+        assert reverse('pay') == '/pay/'
+        assert reverse('create-payment') == '/api/v0/payment/'
```
Comment (noisecapella) [inline bootcamp/urls_test.py]: We don't have other tests that say what the URL name resolves to

**Comment by giocalitri** [comment]
> I cant resist , is this "Klass" and not "class" is suppose to be some kind of a joke ? :-)
 I think someone told me it was called like this

File: ecommerce/api.py (hunk 2 above)
Comment (giocalitri) [inline ecommerce/api.py]: yeah... but a classmethod would be the same, I guess... feel free to leave as is anyway

**Comment by noisecapella** [comment]
@Ferdi It's the term they use in their database. I guess they used `klass` because `class` is often a reserved keyword

**Comment by noisecapella** [comment]
Comments addressed

**Comment by giocalitri** [comment]
I do not see the new code

**Comment by noisecapella** [comment]
Forgot to push. It should be there now

**Comment by giocalitri** [comment]
👍

**Comment by Ferdi** [comment]
@noisecapella ah, ok thanks ! :-)

**Pull request merged**